analyses/sample-distribution-analysis/00-README.md

```diff
@@ -0,0 +1,31 @@
+# OpenPBTA sample-distribution-analysis
+
+## Folder content
+This folder contains scripts tasked to analyze the distribution of samples across cancer types and histologies in the PBTA dataset.  
+`00-install-packages.R` is a script written to install required packages and dependencies. This script is sourced in the other scripts.   
+`01-pbta-analysis-5` is a script written to perform the initial analyses on the PBTA dataset (as discussed in [issue #5](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/5)). This script produces TSV files containing the counts and percentages of samples across each unique cancer type, and the types sorted in order of descending expression at each unique brain location. 
```

Comment:
Would `filter-across-types.R` seem better suited? Part of the script does involve filtering across `primary_sites`, but `filtering.R` may be too broad.

Reply:
Yes, `filter-across-types.R` in an `analyses` directory called `sample-distribution-analyses` seems like it would give someone a good idea about what it does.